docs/api/web-contents.md

```diff
@@ -45,6 +45,13 @@ returns `null`.
 Returns `WebContents` | undefined - A WebContents instance with the given ID, or
 `undefined` if there is no WebContents associated with the given ID.
 
+### `webContents.fromFrame(frame)`
+
+* `frame` WebFrameMain
+
+Returns `WebContents` | undefined - A WebContents instance with the given WebFrameMain, or
+`undefined` if there is no WebContents associated with the given WebFrameMain.
```

Comment:
hm... in what situation is there no WebContents for a WebFrameMain?

Reply:
More accurately, this is for when the WebContents has been destroyed. One of the tests in this PR checks this.